src/styles/theme.js

```diff
@@ -210,6 +210,9 @@ export default ({ color, backgroundColor, borderColor, cardColor, dividerColor }
     backgroundColor: cardColor,
     margin: 4,
   },
+  screenPadding: {
```

Comment:
Why did you decide to name the style this way?

Reply:
I named it `screenPadding` as it is padding applied to elements in a screen component, and can be used in the future for any screens that have components that already don't have padding ( Components such as `OptionRow` do, while `ZulipButton` does not)
However if this is too ambiguous, please do suggest a better name and I shall update the commit.

Reply:
I think I prefer the direction of making components contain the padding that's essential to their layout -- like the padding that's already inside `OptionRow`. Then I think this `screenPadding` style item isn't necessary.

Reply:
OK, we still have it for the buttons. I don't like this name either, though.

Following the naming scheme of lots of margin-related styles in this file, let's call this just `padding`. (I don't love that naming scheme, but changing it is for beyond this PR.)

There is an existing `padding` style, which by that naming scheme should be `smallPadding`. Happily, it looks like nothing is using that, so we can just remove it.